### ISSUE TITLE
Libwbxml ships FindLibWbxml2.cmake rather than Config file

### ISSUE THREAD

**Issue by hosiet** [libwbxml/libwbxml#74]
It seems that libwbxml is shipping a Find module in CMake's private dir. This may not be a good practice. The correct way might be shipping a Config file as described in https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html#config-file-packages .

**Comment by bellmich** [comment]
This feels like breaking the whole past build process of users of the library even if the requested change sounds correct. Nevertheless it looks like the whole directory cmake/modules needs a cleanup. So the first step towards fixing this would be cleaning up lots of useless cmake files in cmake/modules.

**Comment by bellmich** [comment]
Finally I had the time to really look into this cmake stuff and started a bigger cleanup. I hope the pull request #91 will fix this issue here too. Any feedback is welcome and I know that the fix took ages. 🙄 Sorry. 🙇

**Comment by bellmich** [comment]
Looks like I have a problem with the GitHub runners which use Ubuntu. Usually they are way more up-to-date than MacOS in terms of tooling. Unfortunately, I cannot access the real error log and I don't have a Linux system at hand right now.

**Comment by hosiet** [comment]
As the error log says:

```
Errors while running CTest
Output from these tests are in: /home/runner/work/libwbxml/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
```

Can we edit `.github/workflows/` so that we use the output-on-failure flag, or just directly `cat LastTest.log`?

**Comment by bellmich** [comment]
Great idea. I only tried to apply the output-on-failure flag via an environment variable in CMake but it doesn't work as GitHub call ctest directly. I try to do it in the file workflows. Thanks a lot for the hint!

**Comment by bellmich** [comment]
The error is now transparent. It looks like I screwd something up with expat. The code in CMakeLists.txt is quite funny. Expat is required but later I test if it is really present.

**Comment by bellmich** [comment]
GitHub explicitly install libexpat1-dev on the runner. This is visible in the log of the step "Install ubuntu-latest dependencies". Later "find_package( Expat REQUIRED)" in CMakeLists.txt is not able to find Expat but returns normally. EXPAT_FOUND is false afterwards. I have no clue where the problem is.

**Comment by bellmich** [comment]
The pull request finally worked. I compensated the buggy FindEXPAT with the use of PkgConfig as Expat comes always with a file expat.pc. Neither elegant nor nice but it works for now.

**Comment by bellmich** [comment]
From my point of view, I see this issue as fixed with pull request #91. I'm going to prepare a fresh release.
